File: fizz-buzz/fizz-buzz.py
class Solution:
    def fizzBuzz(self, number: int) -> List[str]:
        strings_count = []
        for i in range(1, number+1):
            if i % 15 == 0:
                strings_count.append("FizzBuzz")
            elif i % 3 == 0:
                strings_count.append("Fizz")
            elif i % 5 == 0:
                strings_count.append("Buzz")
            else:
                strings_count.append(str(i))
        return strings_count

File: fizz-buzz/fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let final_array = []
    for (let i=1; i<n+1; i++){
        if (i % 15 === 0){
            final_array.push("FizzBuzz")
            }
        else if (i % 5 === 0){
            final_array.push("Buzz")
        }
        else if (i % 3 === 0){
            final_array.push("Fizz")
        }
        else {
            num = i.toString()
            final_array.push(num)
        };
    };
    return final_array
};
